app/components/header_component/view.html.erb

```diff
@@ -1,12 +1,12 @@
 <% if is_signed_in? %>
-  <%= govuk_header navigation_classes: ["govuk-header__navigation--end"] do |header|
-    header.with_product_name(name: t("header.product_name"))
-    header.with_navigation_item(text: t('header.users'), href: list_of_users_path, active: false) if list_of_users_path.present?
+  <%= govuk_header classes: ["app-header", app_header_class_for_environment],  navigation_classes: ["govuk-header__navigation--end"] do |header|
```

Comment:
Where is `app_header_class_for_environment` class defined?

Reply:
In `view.rb`: https://github.com/alphagov/forms-admin/pull/508/files#diff-ac22ef01c268c39ce344f65c3939793ba98084978c87909988d4322090512d6bR25

Reply:
Never mind, I misread it  👍